content/blogposts/2024/coding-on-the-road-with-android-cody.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Coding on the Road with Android and Cody"
+authors:
+  - name: Tino Wening
+    url: https://twitter.com/PriNova75
+  
+publishDate: 2024-02-06T10:00-07:00
+description: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+tags: [blog]
+slug: "coding-on-the-road-with-android-cody"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-on-the-road/coding-on-the-road-with-android-cody-og.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-on-the-road/coding-on-the-road-with-android-cody-og.png
+---
+
+<Alert type="secondary">This is a guest blog post from our community. <a href="https://discord.com/servers/sourcegraph-969688426372825169" target="_blank">Join our Discord</a> to meet the author and connect with other developers passionate about Code Search and AI!</Alert>
+
+**Hey there, fellow Digital Nomads!** Craving that perfect blend of beachside cocktails and coding deadlines? Ever wished your hotel room came with a built-in IDE, not just a mini bar? Well, strap on your walking shoes and ditch the laptop, because we're about to unlock the secret to mobile coding freedom: **Termux and Cody.**
```

Comment:
```suggestion
**Hey there, fellow Digital Nomads!** Craving that perfect blend of beachside cocktails and coding deadlines? Ever wished your hotel room came with a built-in IDE, not just a mini bar? Well, strap on your walking shoes and ditch the laptop, because we're about to unlock the secret to mobile coding freedom: Termux and Cody.
```